### PR TITLE
Add demo data files and fix the concerned tests - Sanity tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add some data files contain the demo data used in the tests
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | npm run sanity-tests

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15634)

### CODE REVIEW

File: tests/puppeteer/campaigns/data/demo/customer.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  defaultAccount: {
+  DefaultAccount: {
```
Comment (SimonGrn) [inline tests/puppeteer/campaigns/data/demo/customer.js:2]: Does it change somewhere else ?

Comment (nesrineabdmouleh) [reply tests/puppeteer/campaigns/data/demo/customer.js:2]: Yes, I have corrected all files affected by my changes ;)

**Review by boubkerbribri — COMMENTED** [review]
Thank you @nesrineabdmouleh

**Review by SimonGrn — APPROVED** [review]
Thanks @nesrineabdmouleh

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**